### PR TITLE
feat(editor): Add support for `loadOptionsDependsOn` to RLC

### PR DESCRIPTION
**Pull request by MiloradFilipovic**

### CODE REVIEW

File: packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue [hunk 1]
```diff
@@ -429,6 +433,18 @@ export default mixins(debounceHelper, workflowHelpers, nodeHelpers).extend({
 				this.$emit('input', { ...this.value, __regex: mode.extractValue.regex });
 			}
 		},
+		dependentParametersValues() {
+			// Reset value if dependent parameters change
+			if (this.value && isResourceLocatorValue(this.value) && this.value.value !== '') {
+				console.log('reset value');
```
Comment (mutdmour) [inline packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue:439]: hmm?

File: packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue [hunk 2]
```diff
@@ -429,6 +433,18 @@ export default mixins(debounceHelper, workflowHelpers, nodeHelpers).extend({
 				this.$emit('input', { ...this.value, __regex: mode.extractValue.regex });
 			}
 		},
+		dependentParametersValues() {
```
Comment (mutdmour) [inline packages/editor-ui/src/components/ResourceLocator/ResourceLocator.vue:436]: where are the tests? 
7VE

**Review by mutdmour — APPROVED** [review]
😍 thank you

**Comment by janober** [comment]
Got released with [`n8n@0.227.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%400.227.0)

**Pull request merged**